static/js/ducks/telescopes.js

```diff
@@ -0,0 +1,26 @@
+import * as API from '../API';
+import store from '../store';
+
+export const FETCH_TELESCOPES = 'skyportal/FETCH_TELESCOPES';
+export const FETCH_TELESCOPES_OK = 'skyportal/FETCH_TELESCOPES_OK';
+
+export const fetchTelescopes = () => (
+  API.GET('/api/telescope', FETCH_TELESCOPES)
+);
+
+
+const reducer = (state={ telescopeList: [] }, action) => {
+  switch (action.type) {
+    case FETCH_TELESCOPES_OK: {
+      const telescopes = action.data;
```

Comment:
No need to change, but consider using a var name that matches the key name below so you can use object literal property shorthand (just a bit more concise/less to track when looking at). Same with observingRuns reducer above. Fine as is, though.

Reply:
Fixed